pages/vi/profiles/gyawaliamit7.md

```diff
@@ -0,0 +1,19 @@
+# Gyawali Amit Profile
+
+| [Linkedin](https://www.linkedin.com/in/gyawaliamit/) | [Twitter](https://twitter.com/gyawaliamit7) | [Instagram](https://www.instagram.com/amit.lfc.gyawali/)  |  [Profile](https://gyawaliamit7.github.io/AmitResume/)  |
+|:-----------------|:-------------|:---------------:|---------------:|
```

Comment:
@gyawaliamit7 There is empty cell created using this tag. Can you make sure it looks good on UI